psql2mysql.py

```diff
@@ -233,6 +316,33 @@ def migrate(self):
             else:
                 LOG.debug("Table '%s' is empty" % table.name)
 
+    def setupTypeDecorators(self, srcTable, targetTable):
+        """
+        Compare the types of all columns in srcTable and targetTable. If
+        they do not match, try to figure out if we have a TypeDecorator
+        configured that could be used for converting the values. If there
+        is one, change the type of the targetTable's columns accordingly.
+
+        FIXME: Currently we ignore case where on TypeDecorator is found
+        optimistically assuming that SQLalchemy will do the right thing,
+        e.g. as for converting from Boolean (PostgreSQL) to TinyInt (MySQL)
+        ideally we should makes this more explicit by adding a proper precheck
+        and defining supported conversion and raising Errors/Warnings if a non
+        supported combination of Types occurs)
+        """
+        srcColumns = srcTable.columns
+
+        for col in srcColumns:
+            targetCol = targetTable.c[col.name]
+            if not isinstance(targetCol.type, col.type.__class__):
+                decorator = lookupTypeDecorator(col.type, targetCol.type)
+                if decorator:
+                    LOG.debug("Converting values in column '%s' from type "
```

Comment:
This might actually deserve `info`

Reply:
True. And fixed now. Sorry I overlooked this after the rebase.